internal/driver/constants.go

```diff
@@ -0,0 +1,63 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+//
+// Copyright (C) 2022 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package driver
+
+const (
+	Command                   = "command"
+	UsbProtocol               = "USB"
+	Path                      = "Path"
+	SerialNumber              = "SerialNumber"
+	CardName                  = "CardName"
+	BasePath                  = "/dev/video"
+	AutoStreaming             = "AutoStreaming"
+	InputIndex                = "InputIndex"
+	UrlRawQuery               = "urlRawQuery"
+	RtspServerHostName        = "RtspServerHostName"
+	DefaultRtspServerHostName = "localhost"
+	RtspTcpPort               = "RtspTcpPort"
+	DefaultRtspTcpPort        = "8554"
+	RtspUriScheme             = "rtsp"
+	Stream                    = "stream"
+	PrefixInput               = "Input"
+	PrefixOutput              = "Output"
+
+	// API route specific to Device Service
+	ApiRefreshDevicePaths = "/refreshdevicepaths"
+
+	// Metadata descriptions
+	DescNotSpecified = "not specified"
+	DescTimePerFrame = "time per frame"
+	DescHighQuality  = "high quality"
+
+	// V4L2 commands
+	VIDIOC_QUERYCAP  = "VIDIOC_QUERYCAP"
+	VIDIOC_G_INPUT   = "VIDIOC_G_INPUT"
+	VIDIOC_ENUMINPUT = "VIDIOC_ENUMINPUT"
+	VIDIOC_G_FMT     = "VIDIOC_G_FMT"
+	VIDIOC_CROPCAP   = "VIDIOC_CROPCAP"
+	VIDIOC_G_PARM    = "VIDIOC_G_PARM"
+	VIDIOC_ENUM_FMT  = "VIDIOC_ENUM_FMT"
```

Comment:
Use more readable names for the constant names. No reason they have to be the same as the cryptic values

Reply:
Submitted a PR for this modification: https://github.com/edgexfoundry-holding/device-usb-camera/pull/14